libmamba/src/solv-cpp/repo.cpp

```diff
@@ -229,6 +235,27 @@ namespace mamba::solv
             // TODO(C++20) fmt::format
             auto ss = std::stringstream();
             ss << "Unable to read repo solv file '" << name() << '\'';
+            if (const char* str = ::pool_errstr(raw()->pool))
+            {
+                ss << ", error was: " << str;
+            }
+            throw std::runtime_error(ss.str());
+        }
+    }
+
+    void ObjRepoView::legacy_read_conda_repodata(const fs::u8path& repodata_file, int flags) const
+    {
+        auto file = CFile::open(repodata_file, "rb");
```

Comment:
Shouldnt `file` be tested here?

Reply:
This is done in the `CFile::open` function.